pkg/templates/manifest.go

```diff
@@ -177,15 +178,15 @@ kind: Service
 metadata:
   labels:
     component: sonobuoy
-    run: sonobuoy-master
-  name: sonobuoy-master
+    sonobuoy-component: aggregator
```

Comment:
The aggregator service is not queried within the Sonobuoy codebase so this label could be updated without impact.